src/scripts/pbs_status.py

```diff
@@ -264,9 +272,10 @@ def get_qstat_location():
 
 job_id_re = re.compile("\s*Job Id:\s([0-9]+)([\w\-\/.]*)")
 exec_host_re = re.compile("\s*exec_host = ([\w\-\/.]+)")
-status_re = re.compile("\s*job_state = ([QRECH])")
-exit_status_re = re.compile("\s*exit_status = (-?[0-9]+)")
-status_mapping = {"Q": 1, "R": 2, "E": 2, "C": 4, "H": 5}
+status_re = re.compile("\s*job_state = ([QREFCH])")
+exit_status_re = re.compile("\s*[Ee]xit_status = (-?[0-9]+)")
+status_mapping = {"Q": 1, "R": 2, "E": 2, "F": 4, "C": 4, "H": 5}
```

Comment:
F means 'finished' (http://www.pbsworks.com/documentation/support/PBSProRefGuide11.1.pdf, page 231 of this manual)